Cargo.toml

```diff
@@ -6,17 +6,25 @@ authors = ["Stan Drozd <drozdziak1@gmail.com>"]
 [dependencies]
 rita = { path = "./rita" }
 
+[features]
+default = ["guac"]
+guac = ["rita/guac"]
+system_alloc = ["rita/system_alloc"]
+
 [workspace]
-members = ["althea_kernel_interface", "bounty_hunter", "settings", "clu", "exit_db", "stats_server"]
+members = ["althea_kernel_interface", "settings", "clu", "exit_db", "stats_server"]
 
 [profile.release]
 opt-level = "z"
-debug=true
 lto = true
 codegen-units = 1
 incremental = false
 
 [patch.crates-io]
+ethereum-types = {git="https://github.com/paritytech/primitives.git"}
```

Comment:
What about pinning this to a specific commit?

Reply:
the punning is done in cargo.lock, and we check that into git, so it should be consistent